src/lib/annotations/annotatorUtil.js

```diff
@@ -288,8 +320,7 @@ export function getAvatarHtml(avatarUrl, userId, userName) {
             .substring(0, 3);
     }
 
-    const index = parseInt(userId, 10) || 0;
-    return `<div class="bp-annotation-profile avatar-color-${index % AVATAR_COLOR_COUNT}">${initials}</div>`.trim();
+    return `<div class="bp-annotation-profile ${getUserColor(userId)}">${initials}</div>`.trim();
```

Comment:
I'd rather we return an HTML node instead of text (avoid using `innerHTML`)

Reply:
I'm going to add this to a different ticket to do after because it requires a refactoring of how we create the entire annotation element